CAM2CameraDatabaseAPIClient/client.py

```diff
@@ -204,6 +212,11 @@ def update_owner(self, clientID, owner):
         str
             Success message.
 
+        Example
+        -------
+
+            client.update_owner(self.clientID, 'testowner')
```

Comment:
usually this method will be used to: `webUI_client.update_owner(userClientID, 'userClientOwner')` We might want to add more descriptive explanation and use-case scenario.